opm/simulators/flow/Transmissibility_impl.hpp

```diff
@@ -262,11 +262,11 @@ update(bool global, const TransUpdateQuantities update_quantities,
         centroids_cache_[elemIdx] = centroids_(elemIdx);
     }
 
-    auto harmonicMean = [](const Scalar half1, const Scalar half2)
+    auto harmonicMean = [](const Scalar half1, const Scalar Half2)
```

Comment:
I might consider calling these just `x1` and `x2` instead of mixing upper and lower case.

Reply:
ok.